Cargo.lock

```diff
@@ -3426,7 +3391,7 @@ version = "1.6.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "1f559b464de2e2bdabcac6a210d12e9b5a5973c251e102c44c585c71d51bd78e"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
Unsure if that's a good change. Might be worth to rerun cargo update

Reply:
This is very interesting, it is always this dependency, where this occurs. I have no idea why cargo does that.